packages/core/src/connectors/index.ts

```diff
@@ -33,6 +33,42 @@ export const getConnectorInstanceById = async (id: string): Promise<ConnectorIns
   return { connector, ...found };
 };
 
+const isSmsConnector = (connector: ConnectorInstance): connector is SmsConnector => {
+  return connector.metadata.type === ConnectorType.SMS;
+};
+
+export const getSmsConnectorInstance = async (): Promise<SmsConnector> => {
+  const connectors = await getConnectorInstances();
+  const smsConnector = connectors
+    .filter((connector) => connector.connector?.enabled)
```

Comment:
connector.connector is non-nullabe?

Reply:
can we flatten the connector DB data into the ConnecterInstance type? So just call `connector.enabled`?

Reply:
@simeng-li that is discussed before, @darcyYe can you explain this offline?

Reply:
@gao-sun in @darcyYe 's code, connector.connector is nullable (through, after initialize, is non-nullable)

Reply:
should be non-nullable since we call `getConnectorInstances` every time

Reply:
can add an issue for this

Reply:
LOG-1449